static/js/lierc/app.js

```diff
@@ -370,6 +374,7 @@ var App = function(url, user) {
 
     app.collapse_embeds_pref(panel);
     app.monospace_nicks_pref(panel);
+    app.ignores_pref(panel);
```

Comment:
I think you will want to do this when a connection is created, instead of when a panel is created. And probably pass the connection object instead of a panel object.